templates/oioswift-proxy-server.conf.erb

```diff
@@ -55,9 +55,11 @@ use = egg:swift#account_quotas
 [filter:container-quotas]
 use = egg:swift#container_quotas
 
+<% if @middleware_versionning -%>
 [filter:versioned_writes]
 use = egg:swift#versioned_writes
 allow_versioned_writes = true
+<% end -%>
```

Comment:
To disable it, just set `allow_versioned_writes` to `false`.

Reply:
I've changed it